usaspending_api/etl/management/commands/load_table_from_delta.py

```diff
@@ -0,0 +1,227 @@
+        if temp_dest_table_exists:
+            logger.info(f"{temp_destination_table} exists and recreate argument provided. Dropping first.")
+            # If the schema has changed and we need to do a complete reload, just drop the table and rebuild it
+            clear_table_sql = f"DROP TABLE {temp_destination_table}"
+            with db.connection.cursor() as cursor:
+                cursor.execute(clear_table_sql)
+            logger.info(f"{temp_destination_table} dropped.")
+
+        # Recreate the table if it doesn't exist. Spark's df.write automatically does this but doesn't account for
+        # the extra stuff (indexes, constraints, defaults) which CREATE TABLE X LIKE Y accounts for.
+        # If there is no source_table to base it on, it just relies on spark to make it and work with delta table
+        if source_table or source_schema:
+            if source_table:
+                create_temp_sql = f"""
+                    CREATE TABLE {temp_destination_table} (
+                        LIKE {source_table} INCLUDING DEFAULTS INCLUDING IDENTITY
+                    )
+                """
+            else:
+                create_temp_sql = f"""
+                    CREATE TABLE {temp_destination_table} (
+                        {", ".join([f'{key} {val}' for key, val in source_schema.items()])}
+                    )
+                """
+            with db.connection.cursor() as cursor:
+                logger.info(f"Creating {temp_destination_table}")
+                cursor.execute(create_temp_sql)
+                logger.info(f"{temp_destination_table} created.")
+
+                # Copy over the constraints before indexes
+                # Note: we could of included indexes above (`INCLUDING CONSTRAINTS`) but we need to drop the
+                #       foreign key ones
+                if source_table:
+                    logger.info(f"Copying constraints over from {source_table}")
+                    copy_constraint_sql = make_copy_constraints(
+                        cursor, source_table, temp_destination_table, drop_foreign_keys=True
+                    )
+                    if copy_constraint_sql:
+                        cursor.execute("; ".join(copy_constraint_sql))
+                    logger.info(f"Constraints from {source_table} copied over")
+
+            # Converting unsupported data types on spark to their delta equivalent
+            # We will cast these back to their original intended types afterwards
+            # Only applicable if we're copying down a table that was already in postgres
+            alter_column_sql = []
+            source_columns = []
+            if source_model:
+                source_columns = [(column.name, type(column)) for column in source_model._meta.get_fields()]
+            else:
+                source_columns = list(source_schema.items())
+
+            for column_name, column_type in source_columns:
+                if column_type in SPECIAL_TYPES_MAPPING:
```

Comment:
Similar to a comment below:

> Since this may have the potential of a long wait while it executes, it'd be good to put a log statement in here, before the for loop:
> e.g.
> f"Altering columns back to their postgres-side types for columns: {special_columns}"